test/jdk/java/nio/file/FileStore/Basic.java

```diff
@@ -73,8 +73,13 @@ static <V extends FileAttributeView> void testFileAttributes(Path file,
         FileStore store = Files.getFileStore(file);
         boolean supported = store.supportsFileAttributeView(viewClass);
         assertTrue(store.supportsFileAttributeView(viewName) == supported);
-        boolean haveView = Files.getFileAttributeView(file, viewClass) != null;
-        assertTrue(haveView == supported);
+        // A FileSystemProvider may support many views but some of these attributes
+        // in these views may not supported on all FileStores.
+        // So we only do this check on AIX added by JDK-8360887/8361183
+        if (Platform.isAix()) {
+            boolean haveView = Files.getFileAttributeView(file, viewClass) != null;
+            assertTrue(haveView == supported);
+        }
```

Comment:
I think we should replace L76-L77 with
```
if (supported) {
    assertNotNull(Files.getFileAttributeView(file, viewClass));
}
```
so no need for anything AIX specific here.